lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -315,8 +319,10 @@ function dosomething_helpers_add_js_onboarding() {
     ]);
 
     $onboardingSettings = [
-        'enabled' => true,
-        'slides' => [],
+      'globalEnabled' => $onboarding_enabled,
```

Comment:
I changed it to onboardingEnabled cause I did it before your comment, I assume thats alright lol